lib/taurus/core/tango/tangoattribute.py

```diff
@@ -811,7 +811,11 @@ def setRange(self, *limits):
             limits = limits[0]
         low, high = limits
         low = Quantity(low)
+        if low.dimensionless:
+            low = Quantity(low, self._units)
```

Comment:
> About the implementation, I agree that maybe is better to use "low.magnitude", but with my Pint version (0.8.dev0) it is valid. I will try with the docker.

Careful: pint accepts it, but it is not doing what you think. See this similar example:


```python
In [20]: q=Q_("1V")
In [21]: q2=Q_(q, "m")
In [22]: q2
Out[22]: <Quantity(1 V, 'meter')>
In [23]: print q2
1 V m

In [24]: q2.units
Out[24]: <Unit('meter')>
In [25]: q2.magnitude
Out[25]: <Quantity(1, 'volt')>
```
i.e. if you do not use `.magnitude`, you are creating a quantity whose magnitude is a *quantity*.

I think this does not make sense (definitely not in our case), but pint is quite liberal in what it accepts as a magnitude (in order to allow you to use arbitrary objects such as numpy arrays, etc) as the underlying magnitude in quantities.